lifelines/fitters/aalen_johansen_fitter.py

```diff
@@ -14,32 +14,51 @@ class AalenJohansenFitter(UnivariateFitter):
     """Class for fitting the Aalen-Johansen estimate for the cumulative incidence function in a competing risks framework.
     Treating competing risks as censoring can result in over-estimated cumulative density functions. Using the Kaplan
     Meier estimator with competing risks as censored is akin to estimating the cumulative density if all competing risks
-    had been prevented. If you are interested in learning more, I (Paul Zivich) recommend the following open-access
```

Comment:
lol ok 👍

Reply:
put your phone number in

Reply:
lol jk

Reply:
Didn't want to get you in trouble with my esoteric paper recommendations... I will put my home address in as well